src/core_plugins/timelion/public/panels/timechart/schema.js

```diff
@@ -168,15 +168,15 @@ module.exports = function timechartFn(Private, config, $rootScope, timefilter, $
 
         let legendScope = $scope.$new();
         function drawPlot(plotConfig) {
+          if (!$('.chart-canvas', $elem).length) $elem.html(template);
+          const canvasElem = $('.chart-canvas', $elem);
 
-          if (!plotConfig || !plotConfig.length) {
-            $elem.empty();
```

Comment:
As far as I could tell, this code-path was only being hit with the `setTimeout`, so if this did happen we'd prolly get an error throw later on in this code when we try to use the `plotConfig`